mkdocs_include_markdown_plugin/event.py

```diff
@@ -54,10 +54,16 @@ def found_include_tag(match):
 
         return text_to_include
 
+    def interpret_escapes(str):
+        if str is None:
+            return None
+        raw = str.encode('latin-1', 'backslashreplace')
+        return raw.decode('unicode_escape')
```

Comment:
Can you just do it in one line?

```python
return value.encode('latin-1', 'backslashreplace').decode('unicode_escape')
```

Reply:
That's how I did it in the first try, PR #3, but due to line length (over 79 characters), that version does not pass your pre-commit "linter" check. So I was forced to split it into two lines. Please advise.

Reply:
OK. So, what about this?

```python
return value.encode(
    'latin-1', 'backslashreplace'
).decode('unicode_escape')
```